src/robusta/core/sinks/incidentio/incidentio_sink.py

```diff
@@ -0,0 +1,115 @@
+import logging
+from typing import Optional, Dict, List, Any
+from urllib.parse import urljoin
+from robusta.core.sinks.incidentio.incidentio_client import IncidentIoClient
+from robusta.core.sinks.incidentio.incidentio_api import AlertEventsApi
+
+import requests
+
+from robusta.core.reporting.base import BaseBlock, Finding, FindingSeverity, Enrichment, Link, LinkType
+from robusta.core.reporting.blocks import (
+    HeaderBlock,
+    JsonBlock,
+    LinksBlock,
+    ListBlock,
+    MarkdownBlock,
+    TableBlock,
+    KubernetesDiffBlock,
+)
+from robusta.core.sinks.sink_base import SinkBase
+from robusta.core.sinks.sink_config import SinkConfigBase
+from robusta.core.sinks.incidentio.incidentio_sink_params import IncidentioSinkParams
+
+
+class IncidentioSink(SinkBase):
+    def __init__(self, sink_config: SinkConfigBase, registry):
+        super().__init__(sink_config.incidentio_sink, registry)
+        self.source_config_id = sink_config.incidentio_sink.source_config_id
+        self.client = IncidentIoClient(
+            base_url=sink_config.incidentio_sink.base_url,
+            token=sink_config.incidentio_sink.token
+        )
+
+    @staticmethod
+    def __to_incidentio_status_type(title: str) -> str:
+        # Map finding title to incident.io status
+        if title.startswith("[RESOLVED]"):
+            return "resolved"
+        return "firing"
+    
+
+    def __send_event_to_incidentio(self, finding: Finding, platform_enabled: bool) -> dict:
+        metadata: Dict[str, Any] = {}
+        links: List[Dict[str, str]] = []
+
+        # Add Robusta links if platform is enabled
+        if platform_enabled:
+            links.append(
+                {
+                    "text": "🔎 Investigate in Robusta",
+                    "href": finding.get_investigate_uri(self.account_id, self.cluster_name),
+                }
+            )
+        else:
+            links.append(
+                {
+                    "text": "🔎 Enable Robusta UI to investigate",
+                    "href": "https://bit.ly/robusta-ui-incidentio",
```

Comment:
Do we really want this else clause? 
I checked, and this link `https://bit.ly/robusta-ui-incidentio` doesn't have anything

Reply:
Fair point, I removed it.